rebar.config

```diff
@@ -10,7 +10,7 @@
         {aeserialization, {git, "https://github.com/aeternity/aeserialization.git",
                            {ref, "179dd41"}}},
         {aesophia, {git, "https://github.com/aeternity/aesophia.git",
-                    {ref, "e94c1f9"}}},
+                    {ref, "cdc7b90"}}},
```

Comment:
This is the commit that is already on master?! Is this not rebased?

Reply:
Shouldn't it be pointing to aesophia master?

Reply:
Ok, it is easier to review if you rebase while you had to do changes anyway. But nevermind.